rpc_jobs/multi_node_aio.yml

```diff
@@ -1,37 +1,145 @@
 - project:
     name: 'Multi-Node-AIO-Jobs'
     series:
+      - kilo:
+          branch: kilo
+          branches: "kilo.*"
+          # There's no kilo branch on [1], but liberty-12.2 branch seems to
+          # work OK.
+          # [1] https://github.com/rcbops-qe/kibana-selenium.git
+          KIBANA_SELENIUM_BRANCH: "liberty-12.2"
+          USER_VARS: |
+            tempest_test_sets: 'scenario defcore api heat_api smoke'
+          DEPLOY_TELEGRAF: "yes"
+      - liberty:
+          branch: liberty-12.2
+          branches: "liberty-.*"
+          USER_VARS: |
+            tempest_test_sets: 'scenario defcore api heat_api smoke'
+          DEPLOY_TELEGRAF: "yes"
       - mitaka:
           branch: mitaka-13.1
           USER_VARS: |
             tempest_test_sets: 'scenario defcore api heat_api smoke'
+          DEPLOY_TELEGRAF: "yes"
       - newton140:
           branch: newton-14.0
           USER_VARS: |
             tempest_test_sets: 'all'
+          DEPLOY_TELEGRAF: "yes"
       - newton141:
           branch: newton-14.1
           USER_VARS: |
             tempest_test_sets: 'all'
+            lxc_container_vg_name: "USE_DIR_NOT_VG"
+            glance_default_store: "swift"
+            neutron_legacy_ha_tool_enabled: true
+          UPGRADE_FROM_REF: "kilo"
+          DEPLOY_TELEGRAF: "yes"
       - master:
           branch: master
           USER_VARS: |
             tempest_test_sets: 'all'
-    context:
+          DEPLOY_TELEGRAF: "yes"
+    image:
       - xenial:
           DEFAULT_IMAGE: "16.04.2"
       - trusty:
           DEFAULT_IMAGE: "14.04.5"
+    action:
+      - deploy:
+          ACTION_STAGES: >-
+            Prepare MaaS,
+            Setup MaaS,
```

Comment:
You probably want to remove the maas stages, and use DEPLOY_MAAS: yes as default for jobs in the job template.